### ISSUE TITLE
Single-Fluffy Fluff Pile Uses Plural Wording

### ISSUE THREAD

**Issue by Shadow8472** [eidalac/FluffyBreeder#37]
> Stress Event
> 
> After a long and stressful day, you close up Fluffy Town and just collapse from exhaustion.
>
> As you lay there Fluffy 11 all form a makeshift 'fluff pile' to try and comfort you.
>
> You feel better! 

Minor nitpick.  I figured it was worth mentioning.

**Comment by eidalac** [comment]
Yeah, that shouldn't fire without picking multiple fluffies, so the text just assumes plural.  Either way an error.

**Comment by Shadow8472** [comment]
If only one gets picked, Fluffy 11 comes over and cuddles.
